common_code/mass_operator.h

```diff
@@ -142,7 +142,7 @@ namespace Mass
           for (unsigned int q=0; q<phi.n_q_points; ++q)
             phi.submit_value (phi.get_value(q), q);
           phi.integrate (true,false);
-          VectorizedArray<Number> local_sum = VectorizedArray<Number>();
+          VectorizedArrayType local_sum = VectorizedArrayType();
```

Comment:
Don't you need to also change the default template argument in `FEEvaluation` (here and elsewhere) to use `VectorizedArrayType`? I don't think this will compile when you enable the version `1`.

Reply:
I have started with BP5, which does not need the mass operator ;) Commit comes soon!